src/test/java/com/softserveinc/ita/CheckoutPageTest.java

```diff
@@ -0,0 +1,66 @@
+package com.softserveinc.ita;
+
+import com.softserveinc.ita.pageobjects.CheckoutPage;
+import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.ProductPage;
+import com.softserveinc.ita.utils.runners.TestRunner;
+import org.testng.annotations.Test;
+
+import static com.softserveinc.ita.repos.CustomerOrderDataRepo.getCustomerOrderData;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CheckoutPageTest extends TestRunner {
+
+    protected HomePage homePage = new HomePage();
+    protected ProductPage productPage = new ProductPage();
+    protected CheckoutPage checkoutPage = new CheckoutPage();
+
+    @Test
+    public void verifyCheckoutPageFunctionalityTest() {
+        homePage.closeAdBanner();
+        var lastProductOfSection = homePage.getLastProductOfSection("Акційні пропозиції");
+
+        assertThat(lastProductOfSection)
+                .as("Last product of section should be opened")
+                .isEqualTo(productPage.getProductName());
+
+        productPage
+                .addProductToBasket()
+                .goToCheckout()
+                .didCheckoutPageOpenCorrectly();
```

Comment:
create a method with boolean return type for verification, that checkout page opened, it could be checked modal (page) title and some necessary element on it

Reply:
and verify it with assert